gfx/video_driver.c

```diff
@@ -258,6 +258,9 @@ struct aspect_ratio_elem aspectratio_lut[ASPECT_RATIO_END] = {
 };
 
 static const video_driver_t *video_drivers[] = {
+#if defined(HAVE_OPENGL_CORE)
```

Comment:
This should probably not be the first option, if the user typos their `video_driver` configuration it will default to `gl_core` instead of the older `gl` video driver which is likely a safer default?

See PR https://github.com/libretro/RetroArch/pull/7121 where this used to be a problem with vulkan instead.

Reply:
Fixed.